tests/Prometheus.Client.Tests/LabelsHelperTests.cs

```diff
@@ -215,6 +215,12 @@ public void ThrowOnEnumLabelName16()
             Assert.Throws<NotSupportedException>(() => LabelsHelper.ToArray(labels));
         }
 
+        [Fact]
```

Comment:
Could you please add similar test for ValueTuple<string> to ensure the consistency between passing labels as string array and as ValueTuple?

Reply:
sure